### PR TITLE
[Improvement][remote] set connect time out

### PR DESCRIPTION
**Pull request by CalvinKirs**
[Improvement][remote] set connect time out

### CODE REVIEW

File: dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/remote/config/NettyClientConfig.java [hunk 1]
```diff
@@ -39,15 +40,20 @@
...
     /**
-     *  receive buffer size
+     * receive buffer size
      */
     private int receiveBufferSize = 65535;
 
+    /**
+     * connect timeout millis
+     */
+    private int connectTimeoutMillis = 3;
```
Comment (Technoboy-) [inline dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/remote/config/NettyClientConfig.java:55]: 3000

**Comment by dailidong** [comment]
3 millis is so small

File: dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/remote/config/NettyClientConfig.java (hunk 1 above)
Comment (CalvinKirs) [reply dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/remote/config/NettyClientConfig.java]: Thank you very much for your suggestion, this is indeed my mistake, 3 millis is so small

**Comment by CalvinKirs** [comment]
> 3 millis is so small

Thank you very much for your suggestion, this is indeed my mistake, 3 millis is so small

**Review by Technoboy- — APPROVED** [review]

**Pull request merged**